assets/stylesheets/bootstrap/mixins/_buttons.scss

```diff
@@ -199,6 +199,10 @@
     background-image: none;
   }
 
+  &:hover:active {
+    background-color: $black;
```

Comment:
Should favour hover `background-color` when both `hover` and `active`

Reply:
Was it going transparent again when clicked?

Reply:
Yep
![Oct-05-2020 11-23-41](https://user-images.githubusercontent.com/5774647/95068508-48096580-06fd-11eb-8988-31a4f0cae8e9.gif)